src/app/projetos/[id]/page.tsx

```
@@ -0,0 +1,184 @@
+              alt={projeto.nome}
+              layout="fill"
+              objectFit="cover"
+              className="object-center"
+            />
+          </div>
+
+          <div className="prose dark:prose-invert max-w-none">
+            <h3 className="text-2xl font-semibold mb-4">Sobre o Projeto</h3>
+            <div
+              className="text-lg text-muted-foreground leading-relaxed"
+              dangerouslySetInnerHTML={{ __html: projeto.descricao }}
```

Biome (2.3.14)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)